changelog/unreleased/4215

```diff
@@ -0,0 +1,6 @@
+Enhancement: Log file sharing allowed within ownCloud Android app
+
+With this improvement, sharing log files to the Owncloud application itself is possible from the logs screen.
```

Comment:
```suggestion
Sharing log files to the ownCloud app itself is now possible from the logs screen.
```
We shouldn't start all the calens files with "With this improvement,...". That's information we can save for te changelog
Also, always respect `ownCloud` name as it is, it's a proper brand name

Reply:
Done